packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -265,13 +278,16 @@ class ScrollButtons extends Component {
 
 		if (isPageDown(keyCode)) {
 			if (focusableScrollButtons) {
+				preventDefault(ev);
 				Spotlight.setPointerMode(false);
 				Spotlight.focus(ReactDOM.findDOMNode(this.nextButtonRef.current)); // eslint-disable-line react/no-find-dom-node
 			} else if (!nextButtonDisabled) {
 				this.onClickNext(ev);
+				preventDefault(ev);
```

Comment:
Is intended to call `preventDefault` _after_ calling back `onClickNext`?

Reply:
No, no real reason. I will move this line up. 👍